pkg/liquidity-mining/contracts/gauges/StakelessGauge.sol

```diff
@@ -76,6 +79,7 @@ abstract contract StakelessGauge is IStakelessGauge, ReentrancyGuard {
         _rate = rate;
         _period = _currentPeriod();
         _startEpochTime = _tokenAdmin.startEpochTimeWrite();
+        max_relative_weight = ABSOLUTE_MAX_RELATIVE_WEIGHT;
```

Comment:
I'd pass this as a constructor argument - otherwise it needs to be manually decreased later

Reply:
Sounds good.
Note that since gauges are actually proxies, that change would affect the `create` method for every factory type.

Reply:
Yes, that makes sense. They should all receive this argument.

Reply:
Updated.
`create` was removed from the interface, since `RewardsOnlyGauge` doesn't need this.